app/assets/javascripts/reactions.js

```diff
@@ -0,0 +1,40 @@
+function changeReactionCount(value, warpableId) {
+  $('#reaction-count-' + warpableId).html(value);
+}
+
+function changeToReacted(warpableId) {
+  $("#reaction-star-" + warpableId)[0].className = "fa fa-star-o";
+}
+
+function changeToUnreacted(warpableId) {
+  $("#reaction-star-" + warpableId)[0].className = "fa fa-star";
+}
+
+function reactOnWarpable(event) {
```

Comment:
Let's call them Images because we're going to rename them anyways! They're run from ImageController, actually!